blockchain/notifications.go

```diff
@@ -119,6 +127,12 @@ type ReorganizationNtfnsData struct {
 	NewHeight int64
 }
 
+// ChainReorgStarted signals the commencement of a chain reorganization.
+type ChainReorgStarted struct{}
```

Comment:
I don't see a reason for this type since it's empty anyway.  Just use `nil` in the ntfn.